docs/apm-components.asciidoc

```diff
@@ -17,13 +17,18 @@ Generally, there are two ways that these four components can work together:
 
 APM agents on edge machines send data to a centrally hosted APM integration:
 
-image::./images/apm-architecture.png[Architecture of Elastic APM]
+image::./images/apm-architecture.png[Architecture of Elastic APM with a central Elastic agent]
 
 Or, APM agents and the APM integration live on edge machines and enroll via a centrally hosted {agent}:
 
-image::./images/apm-architecture-two.png[Architecture of Elastic APM option two]
+image::./images/apm-architecture-two.png[Architecture of Elastic APM with an edge Elastic agent]
 
-Read on to learn more about each of these components!
+In addition, Elastic supports OpenTelemetry:
+
+[subs=attributes+]
+include::./diagrams/apm-otel-architecture.asciidoc[Architecture of Elastic APM with OpenTelemetry]
```

Comment:
@colleenmcginnis any idea why the in-diagram link to opentelemetry doesn't work?

Reply:
When you export the SVG from Figma, click the three dots next to _SVG_ and make sure "Outline text" is _not_ selected. 

<img width="502" alt="Screen Shot 2023-01-03 at 10 45 12 AM" src="https://user-images.githubusercontent.com/10479155/210445297-651b29cd-0a96-41b2-afb2-4996964b49ba.png">

When "Outline text" is selected, it will treat text as purely a vector shape (essentially a list of points and how those points should be connected) rather than a collection of letters. I also tagged you on a PR to update our internal docs with details on exporting and embedding SVGs in our docs.

Reply:
Thank you!

Reply:
That worked perfectly!